insitu/forms.py

```diff
@@ -11,11 +11,35 @@ class Meta:
 
 
 class ProductRequirementForm(ModelForm):
+    product = ModelChoiceField(disabled=True,
```

Comment:
Try to use inheritance here - have a parent ProductRequirementForm and then extend it.
You can extend and add the disabled options, then extend both disabled for the last form.